modules/c3-assisted-installer-overview.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-overview_{context}"]
+= Overview
+
+This section explains how to install {product-title} on {oci-c3-no-rt} by using the {ai-full}. 
+
+For an alternative installation method, see 'Installing a cluster on {oci-c3} by using the Agent-based Installer'.
+
+.Preinstallation considerations
+
+* Ensure that your installation meets the prerequisites specified for Oracle. For details, see link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP[Access and Considerations] in the Oracle documentation.
+
+* Ensure that your infrastructure is certified and uses a compatible cloud instance type. For details, see link:https://catalog.redhat.com/cloud/detail/216977[Oracle Cloud Infrastructure].
+
+* Ensure that you are performing the installation on a virtual machine.
+
+.Installation process
+
+The installation process builds a `bastion`  host within the designated compartment of the {product-title} cluster. The `bastion` host is used to run two Terraform scripts:
```

Comment:
```suggestion
The installation process builds a `bastion` host within the designated compartment of the {product-title} cluster. The `bastion` host is used to run two Terraform scripts:
```
Is there a reason why "bastion" is in backticks here, but not in backticks later in the content?